frontend/src/scenes/plugins/plugin/PluginCard.tsx

```diff
@@ -124,32 +119,25 @@ export function PluginCard({
                     ) : null}
                     {unorderedPlugin ? (
                         <Tooltip title="This app does not do any processing in order." placement="topRight">
-                            <Col>
-                                <Tag color="#555">-</Tag>
-                            </Col>
+                            <Tag color="#555">-</Tag>
```

Comment:
always tricky with these colors but we can use `LemonTag` and either choose a close color or extend the `type` prop on it

https://storybook.posthog.net/?path=/story/lemon-ui-lemon-tag--lemon-tag